sdk/eventhub/Microsoft.Azure.EventHubs/tests/ServiceFabricProcessor/CheckpointingTests.cs

```diff
@@ -112,7 +112,7 @@ public void CheckpointEventTest()
 
             // EXPECTED RESULT: Normal processing. This case checkpoints specific events. Validate that the
             // last event processed has a higher sequence number than the checkpointed event.
-            Assert.True(state.Processor.TotalErrors == 0, $"Errors found {state.Processor.TotalErrors}");
+            Assert.True(state.Processor.TotalErrors == 0, $"Errors found {state.Processor.TotalErrors}. Last Error: {state.Processor?.LastError?.Message}");
```

Comment:
These seem to be intermixed and not part of the load balancing scope.

Reply:
That was a change I added because this unrelated test failed once on CI and I thought logging a bit more detail may help track it down when it randomly fails in the future.

Reply:
Fair enough;  the SF processor tests tend to be somewhat unstable.   Please feel free to tag them to be skipped and use #7472 as the tracking item.    The Event Hubs team owns that package and have plans to rework the test approach in the future to try and make those more deterministic.